java/test/jmri/implementation/LightControlTest.java

```diff
@@ -21,22 +21,22 @@
 
     @Test
     public void testCtor() {
-        LightControl l = new LightControl();
-        Assert.assertNotNull("LightControl not null", l);
+        lc = new LightControl();
+        Assert.assertNotNull("LightControl not null", lc);
     }
 
     @Test
     public void testCLighttor() {
-        Light o = new AbstractLight("IL1", "test light") {
+        l = new AbstractLight("IL1", "test light") {
```

Comment:
This light can probably be treated just like the light control.

Reply:
Thanks @pabender , good suggestions,
Steve.